docs/activities.md

```diff
@@ -39,6 +39,22 @@ Activity Definitions are named and referenced in code by their <a class="tdlp" h
 
 <div class="tdiw"><div class="tditw"><p class="tdit">Activity Definition</p></div><div class="tdiiw"><img class="tdi" src="/diagrams/activity-definition.svg" alt="Activity Definition" /></div></div>
 
+#### Idempotency
+
+Temporal recommends that Activities are idempotent.
+
+Activities are idempotent if multiple applications of that operation do not change the state of the system beyond the initial application.
+
+Activities may be retried repeatedly, so you may need to use idempotency keys for critical side effects.
+
+Unlike determinism in a Workflow, which actually is a requirement, Temporal cannot enforce that your Activity is idempotent.
```

Comment:
In many cases, Temporal cannot enforce that your workflow is deterministic. Activity idempotency and workflow determinism should be treated with the same importance IMO. No need for this contrast.